x/gamm/keeper/pool_service_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var (
 	defaultSwapFee    = sdk.MustNewDecFromStr("0.025")
-	defaultExitFee    = sdk.MustNewDecFromStr("0.025")
+	defaultExitFee    = sdk.ZeroDec()
```

Comment:
nit: can we just define a `defaultZeroExitFee` in the keeper_test file as a global var and use it everywhere?